distro.py

```diff
@@ -925,16 +925,26 @@ def _lsb_release_info(self):
         Returns:
             A dictionary containing all information items.
         """
-        if not self.include_lsb:
+        cmd = ['lsb_release', '-a']
+        process = subprocess.Popen(
```

Comment:
This doesn't seem to pipe to devnull anymore  and I can't find anywhere where stderr is used meaningfully unless we get an error. Can we move the processing of stderr until that one error branch? (Or better yet: pipe it to /dev/null and not process it at all!)